echo/index.js

```diff
@@ -181,13 +181,32 @@ function Client() {
     debug('request options', requestOptions);
 
     request.post(requestOptions, function onResp(err, response, body) {
-      var statusCode = response && response.statusCode
-        ? response.statusCode
-        : 0;
-
-      if (err || parseInt(statusCode / 100) !== 2) {
-        error('[echoClient] addEvents error', { err: err, body: body, statusCode: statusCode });
-        callback(err || 'error response status code: ' + statusCode);
+      if (err) {
```

Comment:
Unless I don't understand something, which is likely, there are two if/if else bodies which are duplicated, which wasn't the case before. In both situations when there is either a error or a non successful status code, we want the same behavior.

I can see why we want the change the first callback argument to a object, but I don't understand why we would duplicate the code.